packages/data-stores/src/i18n/test/resolvers.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import { getLocalizedLanguageNames } from '../resolvers';
+import { setLocalizedLanguageNames } from '../actions';
+
+describe( 'i18n resolvers', () => {
+	describe( 'getLocalizedLanguageNames', () => {
+		it( 'should set the localized language names', () => {
+			const iter = getLocalizedLanguageNames( 'en-us' );
+
+			expect( iter.next().value ).toEqual( {
```

Comment:
In addition to checking the `value`, should we also be checking the `done` on every round?